app/gui2/src/components/GraphEditor.vue

```diff
@@ -155,12 +155,50 @@ const groupColors = computed(() => {
 })
 
 const currentInteraction = ref<Interaction>()
-class EditingNode extends Interaction {
+class EdgeDragging extends Interaction {
+  nodeId: ExprId
+
+  constructor(nodeId: ExprId) {
+    super()
+    this.nodeId = nodeId
+  }
+
   cancel() {
     componentBrowserVisible.value = false
```

Comment:
Previously this was part of the `EditingNode` interaction. I don't see how closing the CB would be a normal part of cancelling an `EdgeDragging` operation.

Reply:
Removed.